tests/base_scraper/test_correct_scraper.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import unittest
+import urllib
+
+import mozfile
+import mozhttpd_base_test as mhttpd
+from mozprocess import processhandler
+
+
+class CorrectScraperTest(mhttpd.MozHttpdBaseTest):
+    """Test mozdownload for correct choice of scraper"""
+
+    def test_scraper(self):
+        """Testing various download scenarios for DailyScraper"""
+
+        # This works
+        # Now choose four distinctly named packages
+        # Download them and check if correct file name
+        # Then delete again for each cycle
+
+        # Once whole test is written, fix manifest.ini's again.
+
+        tests = [
+            # ReleaseScraper
+            {'command': ['./mozdownload/scraper.py', '-v', 'latest',
+                         '--base_url=%s' % self.wdir,
+                         '--destination=%s' % self.temp_dir],
```

Comment:
The command, base url, and destination is always the same. I would hardcode it in the test, and extend the arguments as necessary.